### ISSUE TITLE
[Bug]: [nightly] Panic occurs reporting "Failed to create consumer by-dev-rootcoord-dml_48" during flush

### ISSUE THREAD

**Issue by binbinlv** [milvus-io/milvus#17530]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 440bce9
- Deployment mode(standalone or cluster): cluster
- SDK version(e.g. pymilvus v2.0.0rc2): pymilvus dev 69
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Panic occurs reporting "Failed to create consumer by-dev-rootcoord-dml_48" during flush:
```
mdp-587-n-milvus-datacoord-5bd8cdb979-r2znf_milvus-ci_datacoord-06c98568778f02f95f95c464236ecda64aadd2c71419c32cef243bccee80e769.log:[2022/06/11 19:11:40.043 +00:00] [FATAL] [logutil.go:134] [panic] [recover="\"Failed to create consumer by-dev-rootcoord-dml_48, error = All attempts results:\\nattempt #1:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #2:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #3:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #4:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #5:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #6:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #7:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #8:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #9:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #10:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #11:server error: ConsumerBusy: Exclusive consumer is already connected\\n\""] [stack="github.com/milvus-io/milvus/internal/util/logutil.LogPanic\n\t/go/src/github.com/milvus-io/milvus/internal/util/logutil/logutil.go:134\nruntime.gopanic\n\t/usr/local/go/src/runtime/panic.go:965\ngithub.com/milvus-io/milvus/internal/mq/msgstream.(*mqMsgStream).AsConsumerWithPosition\n\t/go/src/github.com/milvus-io/milvus/internal/mq/msgstream/mq_msgstream.go:174\ngithub.com/milvus-io/milvus/internal/mq/msgstream.(*mqMsgStream).AsConsumer\n\t/go/src/github.com/milvus-io/milvus/internal/mq/msgstream/mq_msgstream.go:131\ngithub.com/milvus-io/milvus/internal/mq/msgstream.UnsubscribeChannels\n\t/go/src/github.com/milvus-io/milvus/internal/mq/msgstream/msgstream_util.go:35\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).CleanupAndReassign\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:781\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).processAck\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:621\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).watchChannelStatesLoop\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:655"]
mdp-587-n-milvus-datacoord-5bd8cdb979-r2znf_milvus-ci_datacoord-1ec4b7103808e14df08c854e5f9569a561a6d4b8b0aa3fb8eab51a07ba974082.log:[2022/06/11 19:15:33.591 +00:00] [FATAL] [logutil.go:134] [panic] [recover="\"Failed to create consumer by-dev-rootcoord-dml_48, error = All attempts results:\\nattempt #1:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #2:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #3:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #4:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #5:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #6:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #7:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #8:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #9:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #10:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #11:server error: ConsumerBusy: Exclusive consumer is already connected\\n\""] [stack="github.com/milvus-io/milvus/internal/util/logutil.LogPanic\n\t/go/src/github.com/milvus-io/milvus/internal/util/logutil/logutil.go:134\nruntime.gopanic\n\t/usr/local/go/src/runtime/panic.go:965\ngithub.com/milvus-io/milvus/internal/mq/msgstream.(*mqMsgStream).AsConsumerWithPosition\n\t/go/src/github.com/milvus-io/milvus/internal/mq/msgstream/mq_msgstream.go:174\ngithub.com/milvus-io/milvus/internal/mq/msgstream.(*mqMsgStream).AsConsumer\n\t/go/src/github.com/milvus-io/milvus/internal/mq/msgstream/mq_msgstream.go:131\ngithub.com/milvus-io/milvus/internal/mq/msgstream.UnsubscribeChannels\n\t/go/src/github.com/milvus-io/milvus/internal/mq/msgstream/msgstream_util.go:35\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).CleanupAndReassign\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:781\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).processAck\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:621\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).watchChannelStatesLoop\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:655"]
mdp-587-n-milvus-datacoord-5bd8cdb979-r2znf_milvus-ci_datacoord-c7c8715bf4447ee8467df2fdb3a5f4de1100c3610e13f30b45befd3dd952f646.log:[2022/06/11 19:19:44.568 +00:00] [FATAL] [logutil.go:134] [panic] [recover="\"Failed to create consumer by-dev-rootcoord-dml_48, error = All attempts results:\\nattempt #1:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #2:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #3:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #4:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #5:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #6:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #7:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #8:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #9:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #10:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #11:server error: ConsumerBusy: Exclusive consumer is already connected\\n\""] [stack="github.com/milvus-io/milvus/internal/util/logutil.LogPanic\n\t/go/src/github.com/milvus-io/milvus/internal/util/logutil/logutil.go:134\nruntime.gopanic\n\t/usr/local/go/src/runtime/panic.go:965\ngithub.com/milvus-io/milvus/internal/mq/msgstream.(*mqMsgStream).AsConsumerWithPosition\n\t/go/src/github.com/milvus-io/milvus/internal/mq/msgstream/mq_msgstream.go:174\ngithub.com/milvus-io/milvus/internal/mq/msgstream.(*mqMsgStream).AsConsumer\n\t/go/src/github.com/milvus-io/milvus/internal/mq/msgstream/mq_msgstream.go:131\ngithub.com/milvus-io/milvus/internal/mq/msgstream.UnsubscribeChannels\n\t/go/src/github.com/milvus-io/milvus/internal/mq/msgstream/msgstream_util.go:35\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).CleanupAndReassign\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:781\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).processAck\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:621\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).watchChannelStatesLoop\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:655"]
mdp-587-n-milvus-datacoord-5bd8cdb979-r2znf_milvus-ci_datacoord-ed84ab3e36ec35979c56297c08416ab76b1ec905f207745d9280bd814ae9905a.log:[2022/06/11 19:24:00.212 +00:00] [FATAL] [logutil.go:134] [panic] [recover="\"Failed to create consumer by-dev-rootcoord-dml_48, error = All attempts results:\\nattempt #1:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #2:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #3:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #4:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #5:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #6:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #7:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #8:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #9:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #10:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #11:server error: ConsumerBusy: Exclusive consumer is already connected\\n\""] [stack="github.com/milvus-io/milvus/internal/util/logutil.LogPanic\n\t/go/src/github.com/milvus-io/milvus/internal/util/logutil/logutil.go:134\nruntime.gopanic\n\t/usr/local/go/src/runtime/panic.go:965\ngithub.com/milvus-io/milvus/internal/mq/msgstream.(*mqMsgStream).AsConsumerWithPosition\n\t/go/src/github.com/milvus-io/milvus/internal/mq/msgstream/mq_msgstream.go:174\ngithub.com/milvus-io/milvus/internal/mq/msgstream.(*mqMsgStream).AsConsumer\n\t/go/src/github.com/milvus-io/milvus/internal/mq/msgstream/mq_msgstream.go:131\ngithub.com/milvus-io/milvus/internal/mq/msgstream.UnsubscribeChannels\n\t/go/src/github.com/milvus-io/milvus/internal/mq/msgstream/msgstream_util.go:35\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).CleanupAndReassign\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:781\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).processAck\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:621\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).watchChannelStatesLoop\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:655"]
mdp-587-n-milvus-datacoord-5bd8cdb979-r2znf_milvus-ci_datacoord-edd2f7f982bc8d362ac8b77ed5ebc8ad5859c206c018495a0b71bea50a0e1459.log:[2022/06/11 19:28:38.119 +00:00] [FATAL] [logutil.go:134] [panic] [recover="\"Failed to create consumer by-dev-rootcoord-dml_48, error = All attempts results:\\nattempt #1:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #2:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #3:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #4:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #5:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #6:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #7:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #8:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #9:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #10:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #11:server error: ConsumerBusy: Exclusive consumer is already connected\\n\""] [stack="github.com/milvus-io/milvus/internal/util/logutil.LogPanic\n\t/go/src/github.com/milvus-io/milvus/internal/util/logutil/logutil.go:134\nruntime.gopanic\n\t/usr/local/go/src/runtime/panic.go:965\ngithub.com/milvus-io/milvus/internal/mq/msgstream.(*mqMsgStream).AsConsumerWithPosition\n\t/go/src/github.com/milvus-io/milvus/internal/mq/msgstream/mq_msgstream.go:174\ngithub.com/milvus-io/milvus/internal/mq/msgstream.(*mqMsgStream).AsConsumer\n\t/go/src/github.com/milvus-io/milvus/internal/mq/msgstream/mq_msgstream.go:131\ngithub.com/milvus-io/milvus/internal/mq/msgstream.UnsubscribeChannels\n\t/go/src/github.com/milvus-io/milvus/internal/mq/msgstream/msgstream_util.go:35\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).CleanupAndReassign\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:781\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).processAck\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:621\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).watchChannelStatesLoop\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:655"]
mdp-587-n-milvus-datanode-6d884f4486-5l8r2_milvus-ci_datanode-281cfb77c53141b19b662d6702857e5197c23faadca1befe60379cb3a03240d4.log:panic: All attempts results:
mdp-587-n-milvus-datanode-6d884f4486-9xbkr_milvus-ci_datanode-c60f0d712f1f9c8ddf1a6a7e3f6648accee02ee7f5574930307b3f7c37582694.log:panic: All attempts results:
```

### Expected Behavior

No panic

### Steps To Reproduce

```markdown
Running nightly: [pipeline cluster pulsar]

https://ci.milvus.io:18080/jenkins/blue/organizations/jenkins/milvus-nightly-ci/detail/master/587/pipeline
```

### Milvus Log

[artifacts-milvus-distributed-pulsar-master-587-pymilvus-e2e-logs.tar.gz](https://ci.milvus.io:18080/jenkins/job/milvus-nightly-ci/job/master/587/artifact/artifacts-milvus-distributed-pulsar-master-587-pymilvus-e2e-logs.tar.gz)

**Comment by BUPTAnderson** [comment]
@czs007 @yanliang567 
I also encountered this problem in the version-2.0.2, the data-coord node could not start.

data-coord log:
[datacoord-52.log.tar.gz](https://github.com/milvus-io/milvus/files/8896146/datacoord-52.log.tar.gz)

**Comment by BUPTAnderson** [comment]
Every time I restart datacoord, it reports the following error and then exits.
```
[2022/06/14 11:35:27.431 +08:00] [FATAL] [logutil.go:102] [panic] [recover="\"Failed to create consumer by-dev-rootcoord-dml_98, error = All attempts results:\\nattempt #1:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #2:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #3:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #4:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #5:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #6:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #7:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #8:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #9:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #10:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #11:server error: ConsumerBusy: Exclusive consumer is already connected\\n\""] [stack="github.com/milvus-io/milvus/internal/util/logutil.LogPanic\n\t/opt/milvus/internal/util/logutil/logutil.go:102\nruntime.gopanic\n\t/usr/local/go/src/runtime/panic.go:965\ngithub.com/milvus-io/milvus/internal/msgstream.(*mqMsgStream).AsConsumerWithPosition\n\t/opt/milvus/internal/msgstream/mq_msgstream.go:165\ngithub.com/milvus-io/milvus/internal/msgstream.(*mqMsgStream).AsConsumer\n\t/opt/milvus/internal/msgstream/mq_msgstream.go:131\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).unsubscribe\n\t/opt/milvus/internal/datacoord/channel_manager.go:369\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).cleanUpAndDelete\n\t/opt/milvus/internal/datacoord/channel_manager.go:607\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).processAck\n\t/opt/milvus/internal/datacoord/channel_manager.go:578\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).watchChannelStatesLoop\n\t/opt/milvus/internal/datacoord/channel_manager.go:661"]
```

**Comment by binbinlv** [comment]
Set it as urgent for it exists in both 2.0.2 and master.

**Comment by BUPTAnderson** [comment]
I have merge the patch, the datacoord crash again. @xiaofan-luan 
```
[2022/06/15 19:42:22.350 +08:00] [FATAL] [logutil.go:134] [panic] [recover="\"Failed to create consumer by-dev-rootcoord-dml_101, error = All attempts results:\
attempt #1:server error: ConsumerBusy: Exclusive consumer is already connected\
attempt #2:server error: ConsumerBusy: Exclusive consumer is already connected\
attempt #3:server error: ConsumerBusy: Exclusive consumer is already connected\
attempt #4:server error: ConsumerBusy: Exclusive consumer is already connected\
attempt #5:server error: ConsumerBusy: Exclusive consumer is already connected\
attempt #6:server error: ConsumerBusy: Exclusive consumer is already connected\
attempt #7:server error: ConsumerBusy: Exclusive consumer is already connected\
attempt #8:server error: ConsumerBusy: Exclusive consumer is already connected\
attempt #9:server error: ConsumerBusy: Exclusive consumer is already connected\
attempt #10:server error: ConsumerBusy: Exclusive consumer is already connected\
attempt #11:server error: ConsumerBusy: Exclusive consumer is already connected\
\""] [stack="github.com/milvus-io/milvus/internal/util/logutil.LogPanic
/opt/milvus/internal/util/logutil/logutil.go:134
runtime.gopanic
/usr/local/go/src/runtime/panic.go:965
github.com/milvus-io/milvus/internal/mq/msgstream.(*mqMsgStream).AsConsumerWithPosition
/opt/milvus/internal/mq/msgstream/mq_msgstream.go:174
github.com/milvus-io/milvus/internal/mq/msgstream.(*mqMsgStream).AsConsumer
/opt/milvus/internal/mq/msgstream/mq_msgstream.go:131
github.com/milvus-io/milvus/internal/mq/msgstream.(*PmsFactory).NewMsgStreamDisposer.func1
/opt/milvus/internal/mq/msgstream/mq_factory.go:97
github.com/milvus-io/milvus/internal/mq/msgstream.UnsubscribeChannels
/opt/milvus/internal/mq/msgstream/msgstream_util.go:30
github.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).CleanupAndReassign
/opt/milvus/internal/datacoord/channel_manager.go:782
github.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).processAck
/opt/milvus/internal/datacoord/channel_manager.go:621
github.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).watchChannelStatesLoop
/opt/milvus/internal/datacoord/channel_manager.go:656"]
```

**Comment by xiaofan-luan** [comment]
> I have merge the pr #17562 , the datacoord crash again. @xiaofan-luan
> 
> ```
> [2022/06/15 19:42:22.350 +08:00] [FATAL] [logutil.go:134] [panic] [recover="\"Failed to create consumer by-dev-rootcoord-dml_101, error = All attempts results:\
> attempt #1:server error: ConsumerBusy: Exclusive consumer is already connected\
> attempt #2:server error: ConsumerBusy: Exclusive consumer is already connected\
> attempt #3:server error: ConsumerBusy: Exclusive consumer is already connected\
> attempt #4:server error: ConsumerBusy: Exclusive consumer is already connected\
> attempt #5:server error: ConsumerBusy: Exclusive consumer is already connected\
> attempt #6:server error: ConsumerBusy: Exclusive consumer is already connected\
> attempt #7:server error: ConsumerBusy: Exclusive consumer is already connected\
> attempt #8:server error: ConsumerBusy: Exclusive consumer is already connected\
> attempt #9:server error: ConsumerBusy: Exclusive consumer is already connected\
> attempt #10:server error: ConsumerBusy: Exclusive consumer is already connected\
> attempt #11:server error: ConsumerBusy: Exclusive consumer is already connected\
> \""] [stack="github.com/milvus-io/milvus/internal/util/logutil.LogPanic
> /opt/milvus/internal/util/logutil/logutil.go:134
> runtime.gopanic
> /usr/local/go/src/runtime/panic.go:965
> github.com/milvus-io/milvus/internal/mq/msgstream.(*mqMsgStream).AsConsumerWithPosition
> /opt/milvus/internal/mq/msgstream/mq_msgstream.go:174
> github.com/milvus-io/milvus/internal/mq/msgstream.(*mqMsgStream).AsConsumer
> /opt/milvus/internal/mq/msgstream/mq_msgstream.go:131
> github.com/milvus-io/milvus/internal/mq/msgstream.(*PmsFactory).NewMsgStreamDisposer.func1
> /opt/milvus/internal/mq/msgstream/mq_factory.go:97
> github.com/milvus-io/milvus/internal/mq/msgstream.UnsubscribeChannels
> /opt/milvus/internal/mq/msgstream/msgstream_util.go:30
> github.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).CleanupAndReassign
> /opt/milvus/internal/datacoord/channel_manager.go:782
> github.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).processAck
> /opt/milvus/internal/datacoord/channel_manager.go:621
> github.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).watchChannelStatesLoop
> /opt/milvus/internal/datacoord/channel_manager.go:656"]
> ```

Did you see some error message like failed to clean up subscription?

**Comment by xiaofan-luan** [comment]
My guess is that the pulsar web port is not configured so it fallback to original handle path.
If you follow milvus operator deployment it should be http://host:80 , if your pulsar didn't enable proxy it should be http://host:8080

**Comment by BUPTAnderson** [comment]
@xiaofan-luan yes, i get some `failed to clean up subscriptions` WARN msg
```
[2022/06/15 19:38:03.787 +08:00] [WARN] [mq_factory.go:90] ["failed to clean up subscriptions"] [pulsar=pulsar://hadoop2045.rz.momo.com:6650,hadoop2047.rz.momo.com:6650,hadoop2048.rz.momo.com:6650] [topic=by-dev-rootcoord-dml_24] [subname=by-dev-dataNode-67-433568643670281537] [error="Delete \"http://hadoop2045.rz.momo.com:6650,hadoop2047.rz.momo.com:6650,hadoop2048.rz.momo.com:80/admin/v2/persistent/public/default/by-dev-rootcoord-dml_24/subscription/by-dev-dataNode-67-433568643670281537?force=true\": dial tcp: lookup hadoop2045.rz.momo.com:6650,hadoop2047.rz.momo.com:6650,hadoop2048.rz.momo.com: no such host"]
```

**Comment by xiaofan-luan** [comment]
there is a a config in milvus yaml, pulsar.webport, by default is 80, changed it to 8080 and it should work.

**Comment by xiaofan-luan** [comment]
merged to master, to help on verification

**Comment by binbinlv** [comment]
OK.

**Comment by mostafaghadimi** [comment]
Any update for this issue?

**Comment by xiaofan-luan** [comment]
> Any update for this issue? Since I have found that the latest version is 2.0.2 and no other stable version exists!

should be fixed by master branch, we will wait until 2.1 release
